store/streaming/README.md

```diff
@@ -0,0 +1,64 @@
+# State Streaming Service
```

Comment:
I feel this should live in the docs/ folder

Reply:
Do you think it makes sense to add this to docs/core as part of the core doc progression? Either as the latest doc (order 16) or linked to by store.md (order 6).

Reply:
Imo it should be linked by store.md